tests/preprocessing/test_outliers.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import numpy as np
+
+import ehrapy as ep
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_PATH = f"{CURRENT_DIR}/test_preprocessing"
+
+
+class TestOutliers:
+    def setup_method(self):
+        self.mimic_2_10 = ep.dt.mimic_2()[:10]
```

Comment:
running the tests will download the dataset, no? I'm not a huge fan of this, to be honest; we could reuse some of the existing `create a test anndata object` functions.

Reply:
Yup. scanpy does this as well though. It was easier to write the tests like this :)

We can adapt this to create artificial AnnData objects.